From 2899966243fc0db8f1e7cc075f32d00582dee85f Mon Sep 17 00:00:00 2001
From: Danesh M <daneshm90@gmail.com>
Date: Thu, 28 Jan 2016 12:26:24 -0800
Subject: [PATCH 33/35] Improve app info screen

Move packagename label to newline to avoid jank marquee.

CYNGNOS-1649

Change-Id: I060290510e819096dd1496356d592dcfc81f57f4
---
 res/layout/widget_text_views.xml | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/res/layout/widget_text_views.xml b/res/layout/widget_text_views.xml
index d95599d867..150cd9414e 100644
--- a/res/layout/widget_text_views.xml
+++ b/res/layout/widget_text_views.xml
@@ -17,14 +17,13 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal"
+    android:orientation="vertical"
     android:duplicateParentState="true">
 
     <TextView
         android:id="@+id/widget_text1"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_weight="1"
         android:layout_gravity="fill_horizontal|top"
         android:textAppearance="@android:style/TextAppearance.Material.Body1"
         android:textColor="?android:attr/textColorSecondary"
@@ -35,14 +34,12 @@
 
     <TextView
         android:id="@+id/widget_text2"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_weight="1"
-        android:layout_marginStart="8dip"
         android:layout_gravity="fill_horizontal|top"
         android:textAppearance="@android:style/TextAppearance.Material.Body1"
         android:textColor="?android:attr/textColorSecondary"
-        android:textAlignment="viewEnd"
+        android:textAlignment="viewStart"
         android:singleLine="true"
         android:ellipsize="marquee"
         android:duplicateParentState="true" />
-- 
2.17.1

